Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MoodHistoryFragment.java

```diff
@@ -42,12 +47,29 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         moodEventService = new MoodEventService();
         moodEventService.getMoodHistoryUpdates(this);
 
+        addMood = view.findViewById(R.id.add_mood_fab);
+        addMood.setOnClickListener(new View.OnClickListener() {
```

Comment:
Since this file is pretty clean, we should keep it that way by adding a function onAddClick() and setting it to be the listener via the XML attribute onClick, instead of like this